package.json

```
@@ -1,11 +1,33 @@
 {
   "name": "@quarks-css/quarks",
   "description": "A CSS-in-JS library for applying styles to React components via props",
+  "keywords": [
+    "CSS-in-JS",
+    "styled-components",
+    "React"
```

I think standard for keywords is all lowercase. Also threw in a couple others I noticed other libraries use
```suggestion
  "keywords": [
    "css-in-js",
    "css",
    "styled-components",
    "react",
    "typescript",
    "theme"
```